pconduit/env_vars.go

```diff
@@ -17,4 +17,6 @@ package pconduit
 const (
 	EnvConduitConnectorUtilitiesGRPCTarget = "CONDUIT_CONNECTOR_UTILITIES_GRPC_TARGET"
 	EnvConduitConnectorSchemaToken         = "CONDUIT_CONNECTOR_SCHEMA_TOKEN"
```

Comment:
I just noticed this contains the word `schema`, I'd remove it and leave it as `CONDUIT_CONNECTOR_TOKEN` (or `CONDUIT_CONNECTOR_UTILITIES_TOKEN` if we want to make it specific).

Reply:
Good point, in the code it's always just a token. :+1: